content/tutorial/tutorial.md

```diff
@@ -242,11 +248,11 @@ class Square extends React.Component {
 }
 ```
 
-If you click on a Square now, you should see an alert in your browser.
+यदि आप अभी Square पर क्लिक करते हैं, तो आपको आपके ब्राउज़र में एक अलर्ट दिखना चाहिए।
 
->Note
+>ध्यान दें
 >
->To save typing and avoid the [confusing behavior of `this`](https://yehudakatz.com/2011/08/11/understanding-javascript-function-invocation-and-this/), we will use the [arrow function syntax](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions) for event handlers here and further below:
+>टाइपिंग को बचाने और [`this` के भ्रामक व्यवहार](https://yehudakatz.com/2011/08/11/understanding-javascript-function-invocation-and-this/) से बचने के लिए, हम यहां और आगे के हैंडलर के लिए [एरो फ़ंक्शन सिंटैक्स](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions) का उपयोग करेंगे:
```

Comment:
टाइपिंग को बचाने -> better translation?

Reply:
भ्रामक व्यवहार -> too complex

Reply:
`ज़्यादा टाइपिंग और [`this` के भ्रामक व्यवहार](https://yehudakatz.com/2011/08/11/understanding-javascript-function-invocation-and-this/) से बचने के लिए,` ?

Reply:
> भ्रामक व्यवहार -> too complex

any suggestions for this?

Reply:
भ्रमित करने वाले व्यवहार?